2019-09-08-fda-optimization-dataset-1/README.md

```diff
@@ -0,0 +1,77 @@
+# FDA optimization dataset 1 (`OptimizationDataset`)
+
+Prepared by John Chodera `<john.chodera@choderalab.org>`
+
+The [ZINC15 FDA dataset](http://zinc.docking.org/substances/subsets/fda/) was retrieve in `mol2` format on Sun Sep  8 20:44:34 EDT 2019 via:
+http://zinc.docking.org/substances/subsets/fda.mol2?count=all
+
+Some molecules appear in multiple protonation/tautomeric states.
+The following [oechem filterfile](https://docs.eyesopen.com/toolkits/python/molproptk/filter_files.html) was applied:
+```
+MIN_NUM_HVY 3 "Minimum number of heavy atoms"
+MAX_NUM_HVY 55 "Maximum number of heavy atoms"
+MIN_ROT_BONDS 1 "Minimum number of rotatable bonds"
```

Comment:
Is there a reason why you set the minimum number of rotatable bonds to 1 rather than to zero? I'll try and do a quick check and see if this results in exclusion of any molecules. Otherwise this looks good.